mobify/progressive-web-scaffold
# Astro: Merlins account tab states

This gets rid of the segmentedView if the user is logged in 

## Changes
- trigger app events when the log in state changes

## How to test-drive this PR
- log in to your account
- you should see the segmented view disappear on the top
